README.md

```diff
@@ -3,14 +3,14 @@
 [![NuGet](https://buildstats.info/nuget/PuppeteerSharp)][NugetUrl]
 [![Build status](https://ci.appveyor.com/api/projects/status/pwfkjb0c4jfdo7lc/branch/master?svg=true&pendingText=master&failingText=master&passingText=master)][BuildUrl]
 [![Demo build status](https://ci.appveyor.com/api/projects/status/10g64a4aa0083wgf/branch/master?svg=true&pendingText=demo&failingText=demo&passingText=demo)][BuildDemoUrl]
-[![CodeFactor](https://www.codefactor.io/repository/github/kblok/puppeteer-sharp/badge)][CodeFactorUrl]
+[![CodeFactor](https://www.codefactor.io/repository/github/hardkoded/puppeteer-sharp/badge)][CodeFactorUrl]
 [![Backers](https://opencollective.com/hardkoded-projects/backers/badge.svg)][Backers]
 
-[NugetUrl]: https://www.nuget.org/packages/PuppeteerSharp/
-[BuildUrl]: https://ci.appveyor.com/project/kblok/puppeteer-sharp/branch/master
-[BuildDemoUrl]: https://ci.appveyor.com/project/kblok/puppeteer-sharp-0c8w9/branch/master
-[CodeFactorUrl]: https://www.codefactor.io/repository/github/kblok/puppeteer-sharp
-[Backers]: https://opencollective.com/hardkoded-projects
+[NugetUrl](https://www.nuget.org/packages/PuppeteerSharp/)
+[BuildUrl](https://ci.appveyor.com/project/kblok/puppeteer-sharp/branch/master)
+[BuildDemoUrl](https://ci.appveyor.com/project/kblok/puppeteer-sharp-0c8w9/branch/master)
+[CodeFactorUrl](https://www.codefactor.io/repository/github/kblok/puppeteer-sharp)
+[Backers](https://opencollective.com/hardkoded-projects)
```

Comment:
This is not ok. These are [reference links](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links).

Reply:
how about now?

Reply:
Perfect!